src/Resizebar.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { css } from '@emotion/core'
+
+export const Resizebar = ({ axis, rootSelector, bounds }) => {
+  const checkPosition = e => {
+    const barPosition = axis === 'x' ? e.pageX : window.innerHeight - e.pageY
+
+    if (bounds && (bounds.min >= barPosition || bounds.max <= barPosition))
+      return false
+
+    document.documentElement.style.setProperty(rootSelector, barPosition + 'px')
+  }
+
+  const dragstart = e => {
+    e.preventDefault()
+    window.addEventListener('mousemove', checkPosition)
+    window.addEventListener('mouseup', () =>
+      window.removeEventListener('mousemove', checkPosition)
+    )
+    window.addEventListener('touchmove', checkPosition)
+    window.addEventListener('touchend', () =>
+      window.removeEventListener('touchmove', checkPosition)
+    )
+  }
+
+  return <div css={styles[axis]} onMouseDown={dragstart}></div>
```

Comment:
doesn't seem to work, does `onMouseDown` work properly on touch screens?

Reply:
I'm sorry, my mistake for missing for failing to add an event listener to touch screens there. I normally make sure to test every commit, but I couldn't find a way to preview touchscreen functionality from my desktop computer.

Let me know if it works on touch screen now. Also, if you know of ways to test touchscreen functionality in the future, I'd love your input! Thanks!

Reply:
still doesn't work properly, for now you can just preview it on your phone.

Reply:
I tested the newest commit from my phone, and it works when I disable the screen-width check. I can change the min/max values when reaching smaller screens instead of disabling the resizing option altogether. I may also have to give the ResizeBar more width and height on smaller screens.